src/chainlit/input_widget.py

```diff
@@ -220,3 +191,36 @@ def to_dict(self) -> Dict[str, Any]:
             "tooltip": self.tooltip,
             "description": self.description,
         }
+
+
+@dataclass
+class Tags(InputWidget):
+    """Useful to create an input for an array of strings."""
+
+    type: InputWidgetType = "tags"
+    initial: List[str] = Field(default_factory=lambda: list)
+    values: List[str] = Field(default_factory=lambda: list)
+
+    def __init__(
+        self,
+        key: str or int,
+        label: str,
+        initial: List[str] = None,
+        tooltip: str = None,
+        description: str = None,
+    ) -> None:
+        self.key = key
+        self.label = label
+        if initial is not None:
```

Comment:
Does this if statement do anything? if it is None it will just stay None no?
If it does something, should we also add this check to the other widgets?

Reply:
Nvm looked over the fact that initial is a list in this case